rest/handler.go

```diff
@@ -240,6 +240,15 @@ func (h *handler) logRequestLine() {
 	base.LogTo("HTTP", " #%03d: %s %s%s%s", h.serialNumber, h.rq.Method, base.SanitizeRequestURL(h.rq.URL), proto, as)
 }
 
+func (h *handler) effectiveUsername() string {
+	if h.privs == adminPrivs {
+		return "ADMIN"
+	} else if h.user != nil && h.user.Name() != "" {
+		return h.user.Name()
+	}
+	return ""
+}
+
```

Comment:
How is this different than the existing 'currentEffectiveUserName()'?

Reply:
I hadn't seen that.  Reworked to use `currentEffectiveUserName()`